### DISCUSSION TITLE
How to disable the new v3 spash screen?

### DISCUSSION THREAD

**Discussion by rdbisme** [netdata/netdata#19046] [Q&A] [answered]
Is there a way to disable the new v3 splash screen which really wants to make you login on Cloud, and just go directly to the dashboard? The "skip and use the dashboard anonymously" is behind a javascript function which is annoying since you can not bookmark that and have a permalink.

**Comment by ilyam8** [comment, marked as answer]
Hi, @rdbisme. See https://github.com/netdata/netdata/issues/19007#issuecomment-2474653118.
